hedera-node/src/main/java/com/hedera/services/store/tokens/HederaTokenStore.java

```diff
@@ -466,8 +470,8 @@ private ResponseCodeEnum validateFeeSchedule(List<CustomFeesOuterClass.CustomFee
 		return OK;
 	}
 
-	private boolean signsMatch(long a, long b) {
-		return (a < 0 && b < 0) || (a > 0 && b > 0);
+	private boolean areValidPositiveNumbers(long a, long b) {
+		return (a > 0 && b > 0);
```

Comment:
So we don't allow (-25/-100) any more? Just (25/100)?

Reply:
@qnswirlds  yes we allow only positive numbers